web_ui/src/webworkers/inference-image.worker.ts

```diff
@@ -1,26 +1,15 @@
 // Copyright (C) 2022-2025 Intel Corporation
 // LIMITED EDGE SOFTWARE DISTRIBUTION LICENSE
 
-import { InferenceImage, OpenCVLoader } from '@geti/smart-tools';
-import { expose } from 'comlink';
-import type OpenCVTypes from 'OpenCVTypes';
+import { InferenceImage, opencv, OpenCVLoader } from '@geti/smart-tools';
+import { expose, proxy, ProxyMarked } from 'comlink';
 
 declare const self: DedicatedWorkerGlobalScope;
 
-let opencv: OpenCVTypes;
-
-const waitForOpenCV = async () => {
-    if (opencv) return true;
-
-    opencv = await OpenCVLoader();
-
-    if ('ready' in opencv) {
-        await opencv.ready;
-    }
-
-    return false;
+const createInferenceImage = async (): Promise<ProxyMarked> => {
+    return proxy(new InferenceImage(opencv));
 };
 
-const WorkerApi = { InferenceImage, waitForOpenCV, terminate: self.close };
+const WorkerApi = { InferenceImage: createInferenceImage, loadOpenCV: OpenCVLoader, terminate: self.close };
```

Comment:
Can we rename `InferenceImage` to `createInferenceImage` in a follow up PR?

Reply:
Was fighting between `create` and `init` . perhaps `init` would be better?